src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ *  (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString
+    .split(';')
+    .map(el => el.trim())
+    .filter(el => el.length > 0);
+
+  const stylesObj = {};
+
+  stylesArray.map(style => {
+    let eachPair = style.split(':');
```

Comment:
This can be done in one line but not a big deal for me, as it is more readable

Reply:
I actually didn't think about doing it 
**_const eachPair = style.split(':')
      .map(pair => pair.trim());_**
this way. As I was doing the task in steps. 

I think it is better since it is one less variable declaration. 

Thank you Umut.